CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.0.0-beta2 (unreleased)
+
+* [Breaking] Remove `ulog` and expose `*zap.Logger` directly.
+* [Breaking] Upgrade `zap` to `v1.0.0-rc.2` (now go.uber.org/zap, was
+    github.com/uber-go/zap)
```

Comment:
We haven't needed any migration scripts yet, since beta1 was our first release, but yes, we will probably provide a migration script for as many of (hopefully all?) the API changes between beta1 and beta2